### PR TITLE
Add support root to edit the order (used for root actions)

### PR DESCRIPTION
**Pull request by SudharakaP**
Related to https://github.com/opencollective/opencollective/issues/6641

Front-end PR: https://github.com/opencollective/opencollective-frontend/pull/9418

### CODE REVIEW

**Review by znarf — APPROVED** [review]

File: server/graphql/v2/mutation/PaymentMethodMutations.js [hunk 1]
```diff
@@ -58,7 +58,7 @@ const addCreditCard = {
     checkRemoteUserCanUseOrders(req);
 
     const collective = await fetchAccountWithReference(args.account, { throwIfMissing: true });
-    if (!req.remoteUser?.isAdminOfCollective(collective)) {
+    if (!req.remoteUser?.isAdminOfCollective(collective) && !req.remoteUser?.isRoot()) {
```
Comment (Betree) [inline server/graphql/v2/mutation/PaymentMethodMutations.js:61]: Yes for having support staff update the amount/interval for a contribution or canceling it. 

But why add that to the `addCreditCard` mutation? We definitely don't want support staff to handle credit card numbers and enter them in place of users. Unless I'm missing something?

**Review by Betree — CHANGES_REQUESTED** [review]
Blocking feedback on `addCreditCard` mutation

File: server/graphql/v2/mutation/PaymentMethodMutations.js (hunk 1 above)
Comment (SudharakaP) [reply server/graphql/v2/mutation/PaymentMethodMutations.js]: The reason I added it is once in a while (rarely) I thought that support will need to also change the payment method; like if the user has several credit cards etc. Anyways, I don't think we do this often and can't remember a time this was requested recently so I've removed it.

**Review by Betree — APPROVED** [review]
:+1:

**Pull request merged**